crates/indexer/src/models/move_tables.rs

```diff
@@ -64,7 +66,18 @@ impl TableItem {
                 key: write_table_item.key.to_string(),
                 table_handle: standardize_address(&write_table_item.handle.to_string()),
                 decoded_key: write_table_item.data.as_ref().unwrap().key.clone(),
-                decoded_value: Some(write_table_item.data.as_ref().unwrap().value.clone()),
+                decoded_value: Some(
+                    write_table_item
+                        .data
+                        .as_ref()
+                        .map::<serde_json::Value, _>(|data| data.value.clone())
+                        .unwrap_or_default(),
```

Comment:
This is fine with me. But, did we ever figure out why the Tx `WriteSet` doesn't have data?

Reply:
I spend some time looking at the code and I didn't see any obvious reason. To have a better idea, I think, the easiest is to instrument the Aptos-core project and run it to see when these data are updated.